mailviews/previews.py

```diff
@@ -4,12 +4,7 @@
 from collections import namedtuple
 from email.header import decode_header
 
-try:
-    from django.conf.urls import patterns, include, url
-except ImportError:
-    # Django <1.4 compat
-    from django.conf.urls.defaults import patterns, include, url
-
+import django
```

Comment:
Is used here:
https://github.com/quiqueporta/django-mailviews/blob/533251379c94716ea91ba3d2cef8dde4cafc8bc6/mailviews/previews.py#L85